deployment/helm/synopsys-alert/values.yaml

```diff
@@ -41,6 +41,29 @@ cfssl:
   securityContext: {}
   podSecurityContext: {}
 
+# postgres - configurations for the postgres Pod
+postgres:
+  registry: "docker.io/centos" # override the docker registry at container level
+  isExternal: true # false for running Postgres as a container and true for using External Postgres database
+  host: "localhost" # required only for external postgres, for postgres as a container, it will point to <name>-postgres
+  port: 5432
+  # NOTE: do not change usernames when using postgres as a container; only configure if using external database
+  userUserName: sa
+  userPassword: blackduck
+  databaseName: alertdb
+  # pvc related parameters for postgres container
+  persistentVolumeClaimName: "" # set if you want to create your own PVC (used for migration)
+  claimSize: "5Gi" # PVC claim size
+  storageClass: "" # PVC storage class name
+  volumeName: "" # existing persistent volume name backing the PVC
+  # path to postgresql initialization script
```

Comment:
Is there a missing value here for the postgresql initialization script?

Reply:
we don't have one.  I will remove the comment.  The alert container handles the initialization when it starts up.